Rakefile

```diff
@@ -54,6 +54,11 @@ namespace :shipyard do
     end
   end
 
+  desc 'Publish project to npm as scoped package @codeship/shipyard'
+  task :console do
+    sh 'npm publish --access public'
```

Comment:
Are the files builded by then (e.g. sass 2 css)?

Reply:
They can be. The `sass` files will always be ready to go, but the `css` would need to be compiled first, yes.

Reply:
To be clear though, there's no requirement that the CSS is packaged as the best way to consume all of this will be through the `sass` files anyways.

Reply:
Let's start with `sass`. For some playground or prototyping it would be useful to have compiled versions, but we don't have a use-case for that ATM.